README.md

```diff
@@ -1,2 +1,122 @@
 # flask-monitor
 A Prometheus middleware to add basic but very useful metrics for your Python Flask app.
+
+# Metrics
+
+The only exposed metrics (for now) are the following:
+
+```
+request_seconds_bucket{type,status, method, addr, version, isError, le}
+request_seconds_count{type, status, method, addr, version, isError}
+request_seconds_sum{type, status, method, addr, version, isError}
+response_size_bytes{type, status, method, addr, version, isError}
+dependency_up{name}
```

Comment:
@gutorc92, this is an old Big Brother protocol version. Please update your flask-monitor to adhere to Big Brother Metric Protocol v0.2.0.